### PR TITLE
fix: resolve missing kustomize binary

### PR DESCRIPTION
**Pull request by maxres-fr**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,6 +4,7 @@ go 1.13
 
 require (
 	cloud.google.com/go v0.56.0 // indirect
+	github.com/fatih/color v1.10.0 // indirect
```
Comment (wstrange) [inline go.mod:7]: Is this going to get compiled in to every operator release?  If this is needed for tooling only - should we create a tools/ folder, with its own go.mod, etc.

**Review by wstrange — APPROVED** [review]
OK - but see my comment on tools/ 
Just want to make sure we dont bloat the operator itself with modules that are not needed

File: go.mod (hunk 1 above)
Comment (maxres-fr) [reply go.mod:7]: that's weird, wonder if i installed this by accident

**Pull request merged**